rootfs/scheduler/k8s.py

```diff
@@ -145,8 +153,8 @@ def _get_old_rc(self, name, app_type):
                 break
         if exists:
             return prev_rc
-        else:
```

Comment:
You don't like the explicit `if` -- `else`? I've been writing a lot of Go code so this feels right, but not sure which is more pythonic.

Ok, I've already spent too much time typing about this detail. :+1: